apps/meteor/ee/app/models/server/models/LivechatRooms.js

```diff
@@ -68,27 +68,27 @@ LivechatRooms.prototype.unsetAllOnHoldFieldsByRoomId = function (roomId) {
 	);
 };
 
-LivechatRooms.prototype.unsetPriorityById = function (priorityId) {
+LivechatRooms.prototype.unsetSlaById = function (slaId) {
```

Comment:
Just a heads up about a possible conflict on this file in future since I'm moving all the prototype methods out of this file over here -> https://github.com/RocketChat/Rocket.Chat/pull/27046

Reply:
Yep, sadly that branch hasn't been merged 😢 so will fix the ocnflict on main branch when it is.